src/core/Header/HeaderBreadcrumbs.tsx

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import '@itwin/itwinui-css/css/header.css';
+import SvgChevronRight from '@itwin/itwinui-icons-react/cjs/icons/ChevronRight';
+
+import React from 'react';
+
+import { useTheme } from '../utils/hooks/useTheme';
+
+export type HeaderBreadcrumbsProps = {
+  items: React.ReactNode[];
+};
+
+/**
+ * Header breadcrumbs, adds chevron between each provided items.
+ * @example
+ * <HeaderBreadcrumbs
+ *   items={[
+ *     <HeaderButton key={...} ... />
+ *     <HeaderButton key={...} ... />
+ *   ]}
+ * />
+ */
+export const HeaderBreadcrumbs = (props: HeaderBreadcrumbsProps) => {
+  const { items } = props;
+
+  useTheme();
+  return (
+    <>
```

Comment:
I did some research on accessible breadcrumbs and found that we should use `nav` with `aria-label='breadcrumbs'`. And the active breadcrumb should have `aria-current='page'`. https://www.aditus.io/patterns/breadcrumbs/#technical-considerationg

That's exactly how bootstrap does it too: https://getbootstrap.com/docs/4.0/components/breadcrumb/

Reply:
If we want to add an element containing the breadcrumbs the css will need to be updated as the `.iui-chevron` currently require to be direct descendant of `.iui-left`.

Reply:
Fixed in https://github.com/iTwin/iTwinUI/pull/81 and will be available in next release

Reply:
use css v0.14.0

https://github.com/iTwin/iTwinUI/blob/fbe27813a05cd53cefdfd97a2c45ce2a90a893e1/backstop/tests/header.html#L117

Reply:
Updated the `HeaderBreadcrumbs` and `HeaderButton` components, the strings are localized so I added the `translatedStrings` prop to both components along tests for it (added such a test for `Header` component that was missing for `translatedStrings`) I also added JSDoc to explain where these strings were used to help fill them correctly. (So there is on only this line to review.)

Reply:
Now it's just missing `aria-current=location`

Reply:
I mixed the `page` with `aria-label` instead of `aria-current`, which does not need to be localized at all.

Reply:
> Now it's just missing `aria-current=location`
This is referring to the other comment in `HeaderButton` or it should have that here too ?

Reply:
The other comment. I will close this.